README.md

```diff
@@ -103,16 +106,7 @@ indico
 Batch requests allow you to process larger volumes of data more efficiently by grouping many examples into a single request.  Simply call the batch method that corresponds to the API you'd like to use, and ensure your data is wrapped in an array.
 
 ```javascript
-var indico = require('indico.io')
-
-function fn(err, res) {
-  if (err) {
-    console.log('err: ', err);
-    return;
-  }
-
-  console.log(res):
-}
+var indico = require('indico.io')f
```

Comment:
Omg so rude. I'll fix that.
On Sat, Apr 11, 2015 at 15:14 Madison May <notifications@github.com> wrote:

> In README.md
> <https://github.com/IndicoDataSolutions/IndicoIo-node/pull/24#discussion_r28198919>
> :
>
> > @@ -103,16 +106,7 @@ indico
> >  Batch requests allow you to process larger volumes of data more efficiently by grouping many examples into a single request.  Simply call the batch method that corresponds to the API you'd like to use, and ensure your data is wrapped in an array.
> >
> >  ```javascript
> > -var indico = require('indico.io')
> > -
> > -function fn(err, res) {
> > -  if (err) {
> > -    console.log('err: ', err);
> > -    return;
> > -  }
> > -
> > -  console.log(res):
> > -}
> > +var indico = require('indico.io')f
>
> There's an awkward f hanging out at the end of this line, haha.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/IndicoDataSolutions/IndicoIo-node/pull/24/files#r28198919>
> .
>